core/templates/dev/head/pages/topic-editor-page/modal-templates/tag-misconception-modal.template.html

```diff
@@ -5,17 +5,20 @@ <h3>
 </div>
 <div class="modal-body new-subtopic-title-editor">
   Selected Misconception: <[selectedMisconception.getName()]>
-  <div ng-repeat="misconception in misconceptions track by $index">
-    <button class="btn btn-default" style="width: 100%" ng-click="selectMisconception(misconception)">
-      <[misconception.getName()]>
-    </button>
-    <div ng-if="selectedMisconception.getId() === misconception.getId()">
-      <label class="misconception-label"> Note to creators: </label>
-      <angular-html-bind html-data="misconception.getNotes()">
-      </angular-html-bind>
-      <label class="misconception-label"> Misconception Feedback: </label>
-      <angular-html-bind html-data="misconception.getFeedback()">
-      </angular-html-bind>
+  <[mis]>
```

Comment:
what's this?

Reply:
Oops sorry it was a mistake. Removed it.